src/js/index.js

```diff
@@ -248,16 +248,60 @@ export const hpe = deepFreeze({
     },
   },
   checkBox: {
-    color: 'selected-text',
-    gap: 'small',
-    check: {
-      radius: '2px',
+    hover: {
+      border: {
+        color: undefined,
+      },
+      background: {
+        color: 'background-contrast',
+      },
     },
     border: {
+      color: 'border',
       width: '1px',
     },
+    check: {
+      radius: '2px',
+      extend: ({ theme, checked, indeterminate }) => `
+      background-color: ${
+        checked || indeterminate
+          ? theme.global.colors['green!']
+          : theme.global.colors.background[theme.dark ? 'dark' : 'light']
+      };
+      ${(checked || indeterminate) && 'border: none;'}
+        `,
+    },
+    icon: {
+      size: '18px',
```

Comment:
What are we trying to achieve here? It seems like the icon should scale to its parent.

Reply:
I was trying to make the icon smaller but turns out the size was fine it was the stroke width that was wrong it was `4px` by default and in designs it is `2px`

Reply:
👍